src/lib/edge-functions/registry.js

```diff
@@ -178,6 +184,20 @@ class EdgeFunctionsRegistry {
     return declarations
   }
 
+  static getEnvironmentVariables(envConfig) {
+    const env = Object.create(null)
+    Object.entries(envConfig).forEach(([key, variable]) => {
+      if (variable.sources.includes('ui')) {
```

Comment:
Are you sure we only want to include variables with source of `ui`? Wouldn't it be better to exclude the ones with only `process`? See https://github.com/netlify/cli/blob/main/src/utils/dev.js#L14-L40.

Reply:
I added account and addons.
I tested account and they are available in deno.
For addons I would assume they are, but couldn't find an addon that sets environment variables and works with edge functions. split testing sets envs but edge functions are not triggered.
general & configFile & process is not available in deno